index.md

```diff
@@ -74,4 +74,41 @@ rst2pdf sample.rst
 
 Check for a new file called `sample.pdf` - and you are already on your `rst2pdf` journey!
 
+## Tips, Tricks and Tools
+
+There are a few other PDF tools that we use and love, they are included here in case you need them too!
+
+### Working with PDF Files
+
+**pdftk: The PDF Toolkit** [https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/](https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/)
+
+This commandline tool allows you to combine PDFs (useful for adding coversheets), extract and change metadata such as bookmarks, and much more.
+
+Try this for 12 slides per page, A4 landscape (also ideal for checking that the fonts are large enough, you should be able to read this when printed ...)
```

Comment:
Should lines 87-91 be moved to below line 95 as this example seems to refer to pdfjam, rather than pdftk?

if so, I'll move when merging.